index.html

```diff
@@ -1,45 +1,38 @@
 <!DOCTYPE html>
 <html lang="en">
-<link rel="stylesheet" href="style.css" />
-<link href="https://fonts.googleapis.com/css2?family=Ranchers&display=swap" rel="stylesheet">
-
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-
-<body>
-
-  <h1>
-
-    by <strong>Jacqueline Woodson</strong></h1>
-
-  <p>on paper
-
-
-
-    The first time I write my full name </p>
-
-  Jacqueline Amanda Woodson
-
-  without anybody's help
-  on a clean white page in my composition notebook,
-  I know
-
-  if I wanted to
-
-  I could write anything.
-
-  <em>Letters becoming words, words gathering meaning,
-    becoming
-    thoughts outside my head</em>
-
-  becoming sentences
-
-  written by
-
-  <strong> Jacqueline Amanda Woodson</strong>
-</body>
-
+  <link rel="stylesheet" href="style.css" />
+  <link
+    href="https://fonts.googleapis.com/css2?family=Ranchers&display=swap"
+    rel="stylesheet"
+  />
+
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <h1>by <strong>Jacqueline Woodson</strong></h1>
+
+    <p>on paper The first time I write my full name</p>
+
+    <p><strong> Jacqueline Amanda Woodson</strong></p>
+    <div></div>  
```

Comment:
Going forward, we'll look at ways to achieve design/presentation effects without using empty `div`s 👍🏻